tests/python/unittest/test_gluon_contrib.py

```diff
@@ -228,6 +228,30 @@ def test_sampler():
     assert list(interval_sampler) == [0, 3, 6, 9]
 
 
+def test_deformable_convolution():
+    net = nn.HybridSequential()
+    net.add(
+        Deformable_Convolution(10, kernel_size=(3, 3), strides=1, padding=0),
+        Deformable_Convolution(10, kernel_size=(3, 2), strides=1, padding=0, activation='relu',
+                               offset_use_bias=False, use_bias=False)
```

Comment:
could you add tests for all possible combinations of arguments?

Reply:
Hi, thanks for the reply! Is it enough now?